docs/configuration.md

```diff
@@ -236,6 +236,24 @@ Note: DO NOT USE * configuraion in production deployment.
   cors: "*"
 ```
 
+## Database transactions automatic retry
+
+Gohan supports automatic retry of database transactions that have failed due to
+a database deadlock. Under some circumstances (heavy load, multiple clients)
+it is possible that a deadlock occurs in the database and by default,
+Gohan returns an error via API. To override this behaviour configure
+automatic retry of transactions: set retry count to a value greater than 0
+and optionally configure retry intverval in milliseconds.
+
+```yaml
+  retry_tx:
```

Comment:
it's not clear from config, this is only for deadlocks. Description is fine, but config should also be clear.